sessionctx/variable/sysvar.go

```diff
@@ -360,6 +360,14 @@ func RegisterSysVar(sv *SysVar) {
 	sysVarsLock.Unlock()
 }
 
+// UnregisterSysVar removes a sysvar from the SysVars list
```

Comment:
Only used in the tests? If so, it is best to specify this in the comments.

Reply:
Yes, but that is only the case for another week. I plan to use it in an upcoming PR for Security Enhanced Mode.